MTM-64124 Deprecation note for attaching headers to microservices requests (#3754)
SoftwareAG/c8y-docs: content/change-logs/platform-services/cumulocity-2025-322-0-deprecation-of-tenant-options-in-request-headers.md

---
date: 

> title: Deprecation note for attaching the tenant options as headers to the microservice requests

Reviewer: ```suggestion
title: Deprecating the attachment of tenant options as headers to microservice requests
```

product_area: Platform services
change_type:
  - value: change-inv-3bw8e
    label: Announcement
component:
  - value: component-JlFdtOPva
    label: REST API
build_artifact:
  - value: tc-QHwMfWtBk7
    label: cumulocity
ticket: MTM-64123
version: 2025.322.0
---

> Until now tenant options have been attached to each microservice request. Microservices proxy has added the tenant options to the request headers and forwarded the request to the respective microservice.

Reviewer: ```suggestion
Until now, tenant options were attached to each microservice request. The microservice proxy added the tenant options to the request headers and forwarded the request to the respective microservice.
```

> For example, when creating a tenant option in a category that matches to the application context path, the value has been passed to the microservice by the microservice proxy on the platform as a header (key => value). 

Reviewer: ```suggestion
For example, when creating a tenant option in a category that matches the application context path, the value has been passed to the microservice by the microservice proxy on the platform as a header (key => value). 
```

> This functionality is now deprecated. The tenant options will no longer be attached as headers to the requests to the microservices, and the removal is planned for Q1 2026 in CD versions and in the 2027 annual release.

Reviewer: ```suggestion
This functionality is now deprecated. Tenant options will no longer be attached as headers to requests to microservices. The removal is planned for Q1 2026 in CD versions and in the 2027 annual release.
```
